src/content/upgrades/merge/issuance/index.md

```diff
@@ -15,79 +15,25 @@ The **issuance** of ETH is the process of creating ETH that did not previously e
+## Issuance {#issuance}
 
-[More on The Merge](/upgrades/merge/)
-
-## Pre-merge {#pre-merge}
-
-### Execution layer issuance {#el-issuance-pre-merge}
-
-Under proof-of-work, miners only interact with the execution layer and are rewarded with block rewards if they are the first miner to solve the next block. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward has been 2 ETH per block. Miners are also rewarded for publishing [ommer](/glossary/#ommer) blocks, which are valid blocks that don't end up in the longest/canonical chain. These rewards max out at 1.75 ETH per ommer, and are _in addition to_ the reward issued from the canonical block. Mining is an economically intensive activity, requiring high levels of ETH issuance to sustain.
-
-### Consensus layer issuance {#cl-issuance-pre-merge}
-
-The [Beacon Chain](/history/#beacon-chain-genesis) went live in 2020. Instead of miners, it is secured by validators using proof-of-stake. This chain was bootstrapped by Ethereum users depositing ETH one-way into a smart contract on Mainnet, which the Beacon Chain listens to, crediting the user with an equal amount on the new chain. Until The Merge happens, the Beacon Chain's validators are not processing transactions and are essentially coming to consensus on the state of the validator pool itself.
-
-Validators on the Beacon Chain are rewarded with ETH for attesting to the state of the chain and proposing blocks. Rewards (or penalties) are calculated and distributed at each epoch (every 6.4 minutes) based on validator performance. The validator rewards are **significantly** less than the miner rewards issued on proof-of-work (2 ETH every ~13.5 seconds), as operating a validating node is not an economically intense activity and thus does not require or warrant as high a reward.
-
-### Pre-merge issuance breakdown {#pre-merge-issuance-breakdown}
-
-Total ETH supply: **~119,300,000 ETH** (as of Q2 2022)
-
-**Execution layer issuance:**
-
-- Estimating at 2.08 ETH per 13.3 seconds\*: **~4,930,000** ETH issued in a year
-- Currently inflating at **~4.13%** (4.93M per year / 119.3M total)
-- \*This includes the 2 ETH per canonical block, plus an average of 0.08 ETH over time from ommer blocks. Also uses 13.3 seconds, the baseline block time target without any influence from a [difficulty bomb](/glossary/#difficulty-bomb). ([See source](https://bitinfocharts.com/ethereum/))
+Ethereum validators are rewarded with ETH for attesting to the state of the chain and proposing blocks. Rewards (or penalties) are calculated and distributed at each epoch (every 6.4 minutes) based on validator performance. The validator rewards are **significantly** less than the miner rewards that used to be issued on proof-of-work (2 ETH every ~13.5 seconds), as operating a validating node is not an economically intense activity and thus does not require or warrant as high a reward.
 
 **Consensus layer issuance:**
```

Comment:
Do we need this heading? If we've got rid of execution layer issuance then we no longer have a point of comparison 🤔